explorer/lib/explorer_web/controllers/task_controller.ex

```diff
@@ -0,0 +1,60 @@
+defmodule ExplorerWeb.TaskController do
+  use ExplorerWeb, :controller
+
+  def task(conn, %{"id" => id}) do
+    # Test, returns the name of the ERC20 token
+    # import MyERC20Token
+    # MyERC20Token.get_erc20_name() |> elem(1) |> IO.puts()
+
+    # Returns the EigenLayer AVSDirectory contract.
+    # data = AlignedLayerServiceManager.avs_directory() |> Ethers.call()
+
+    # Returns the AlignedLayer "meaning" value
+    # data = AlignedLayerServiceManager.get_meaning() |> Ethers.call() |> IO.puts()
+
+    # Returns AlignedLayer latestTaskNum
+    # data = AlignedLayerServiceManager.latest_task_num() |> Ethers.call() |> IO.puts()
+
+    # Returns AlignedLayer is_aggregator -> bool
+    # data = AlignedLayerServiceManager.is_aggregator("0x703E7dE5F528fA828f3BE726802B2092Ae7deb2F") |> Ethers.call()
+
```

Comment:
could this be removed?

Reply:
I removed the first one (get_erc20_name), but the rest of the gets will be used in the next explorer iteration (for example get_latest_task_num will be used to show last task)